src/renderer/interfaces/LightMetadata.ts

```diff
@@ -0,0 +1,13 @@
+import REGL = require('regl');
+
+export interface LightMetadata {
+    lightPositions: REGL.Vec3[];
```

Comment:
since this is the data for one light (maybe just call it `Light`?), these properties should be singular, i.e. `lightPosition: REGL.Vec3`

Reply:
Sounds good. 👍